src/util/draws/draw_helper_functions.jl

```diff
@@ -138,6 +170,7 @@ function update_availability!(
     return availability, nexttransition
 end
 
+# TODO: Replace with exponential drawing instead
```

Comment:
Do you mean **_Geometric_** drawing?

Reply:
Geometric would definitely replicate what we have better. I suppose proper exponential draws of the distribution would have to be rounded anyways.